test/RightsService.Business.UnitTests/Commands/Role/GetRoleCommandTests.cs

```diff
@@ -1,43 +1,269 @@
-﻿using LT.DigitalOffice.RightsService.Business.Role.Interfaces;
+﻿using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Exceptions.Models;
+using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+using LT.DigitalOffice.Models.Broker.Responses.User;
+using LT.DigitalOffice.RightsService.Business.Role;
+using LT.DigitalOffice.RightsService.Business.Role.Interfaces;
+using LT.DigitalOffice.RightsService.Data.Interfaces;
+using LT.DigitalOffice.RightsService.Mappers.Interfaces;
 using LT.DigitalOffice.RightsService.Models.Db;
 using LT.DigitalOffice.RightsService.Models.Dto.Models;
+using LT.DigitalOffice.RightsService.Models.Dto.Responses;
+using LT.DigitalOffice.UnitTestKernel;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+using Moq;
 using Moq.AutoMock;
 using NUnit.Framework;
 using System;
 using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace LT.DigitalOffice.RightsService.Business.Commands.Role
 {
     class GetRoleCommandTests
     {
+        private AutoMocker _mocker;
+        private Mock<Response<IOperationResult<IGetUsersDataResponse>>> _operationResultGetUsersData;
```

Comment:
it is not the best practice use autoMocker with usual mock

Reply:
++